projects/storefrontlib/layout/a11y/keyboard-focus/services/select-focus.util.ts

```diff
@@ -57,7 +57,7 @@ export class SelectFocusUtility {
    * @param invisible indicates whether hidden focusable elements should be returned
    */
   findFocusable(
-    host: HTMLElement,
+    host: HTMLElement | null | undefined,
```

Comment:
I removed undefined from the type, and it still works. Why is it here?

Reply:
makes sense! it ties to this here https://github.com/SAP/spartacus/pull/15658#discussion_r870762291 since we're saying to put | undefined instead